### PR TITLE
ADD facilitategovernor.com

### PR DESCRIPTION
**Pull request by FadeMind**
Annoying lots of popups with this domain. Default blocked by ESET.

### CODE REVIEW

File: data/StevenBlack/hosts [hunk 1]
```diff
@@ -3052,4 +3052,7 @@
 0.0.0.0 zipservice.adalliance.io
 
 # Added February 28, 2021
-0.0.0.0 github.com.cnpmjs.org
\ No newline at end of file
+0.0.0.0 github.com.cnpmjs.org
+
+# Added March 15, 2021
+0.0.0.0 facilitategovernor.com
```
Comment (dnmTX) [inline data/StevenBlack/hosts:3058]: @FadeMind you can add those as well,if you want:
`0.0.0.0 www.facilitategovernor.com`
`0.0.0.0 mvideo.facilitategovernor.com`
`0.0.0.0 ns1.facilitategovernor.com`

File: data/StevenBlack/hosts [hunk 2]
```diff
@@ -3056,3 +3056,6 @@
 
 # Added March 15, 2021
 0.0.0.0 facilitategovernor.com
+0.0.0.0 mvideo.facilitategovernor.com
+0.0.0.0 ns1.facilitategovernor.com
+0.0.0.0 www.facilitategovernor.com
```
Comment (FadeMind) [inline data/StevenBlack/hosts:3061]: @dnmTX here. Thanks 

CC @StevenBlack

Comment (StevenBlack) [reply data/StevenBlack/hosts:3061]: Thanks Tomasz @FadeMind and thanks Dan @dnmTX!

**Comment by StevenBlack** [comment]
Merging!

**Pull request merged**